extensions/amp-accordion/1.0/README.md

```diff
@@ -0,0 +1,641 @@
+      <BentoAccordionSection
+        id="section3"
+        key={3}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section3 expanded') : alert('section3 collapsed'}
+        }>
+        <BentoAccordionHeader><h1>Section 3</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 3</BentoAccordionContent>
+      </BentoAccordionSection>
+    </BentoAccordion>
+```
+
+#### Layout and style
+
+**Container type**
+
+The `BentoAccordion` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties). These can be applied inline:
+
+```jsx
+<BentoAccordion style={{width: '300px', height: '100px'}}>
+  ...
+</BentoAccordion>
+```
+
+Or via `className`:
+
+```jsx
+<BentoAccordion className='custom-styles'>
+  ...
+</BentoAccordion>
+```
+
+```css
+.custom-styles {
+  height: 100px;
+  width: 100%;
+}
+```
+
+#### Props
+
+##### BentoAccordion
+
+###### animate
+
+If true, then uses "roll-down" / "roll-up" animation during the expansion and collapse of each section
+Default: `false`
+
+###### as
+
+todo
+
+###### expandSingleSection
+
+If true, then expanding 1 section will automatically collapse all other sections:
+Default: `false`
+
+###### id
+
+todo
+
+##### BentoAccordionSection
+
+###### animate
+
+If true, then uses "roll-down" / "roll-up" animation during the expansion and collapse the section
+Default: `false`
+
+###### as
+
+todo
+
+###### expanded
+
+If true, expands the section.
+Default: `false`
+
+###### id
+
+todo
+
+###### onExpandStateChange
+
+```typescript
+(expanded: boolean): void
+```
+
+Callback to listen for expand state changes. Takes a boolean flag as parameter indicating whether the section was just expanded (`false` indicates it was collapsed)
+
+##### BentoAccordionHeader
+
+###### as
+
+todo
+
+###### class
+
+Class to add to the section header's tag
+
+###### id
+
+todo
+
+###### role
+
+Role to add to the section header's tag.
+Default: `'button'`
+
+###### tabIndex
+
+Tabindex to add to the section header's tag.
+
+##### BentoAccordionContent
+
+###### as
+
+todo
+
+###### class
+
+Class to add to the section header's tag
+
+###### id
+
+todo
+
+###### role
+
+Role to add to the section header's tag.
+Default: `'region'`
+
+#### Styling
+
+You may use the `BentoAccordion` element selector to style the carousel freely.
+
+## Version notes
+
+The experimental `1.0` version of `amp-accordion` does not support session states. It behaves as if the `disable-session-states` attribute is always applied.
+
+Version `0.1` and `1.0` are compatible with `amp-bind`, but some binding syntax is different. You may bind directly with the `expanded` attribute in version `1.0`. The `data-expand` binding is not supported in version `1.0`. See the [`expanded` attribute](#expanded) for further information.
```

Comment:
This section is unneeded because there is no 0.1 version for non-AMP modes

Reply:
removed